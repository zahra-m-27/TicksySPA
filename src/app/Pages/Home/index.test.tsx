import ReactDOM from 'react-dom';
import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';
import HomePage from './index';

const sample = (
  <MemoryRouter>
    <HomePage />
  </MemoryRouter>
);

test('render in ReactDOM', () => {
  const div = document.createElement('div');
  ReactDOM.render(sample, div);
});

it('snapshot', () => {
  const snapshot = renderer.create(sample).toJSON();
  expect(snapshot).toMatchSnapshot();
});
