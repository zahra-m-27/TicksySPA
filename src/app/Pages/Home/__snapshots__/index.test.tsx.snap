// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="container"
>
  <div
    className="container"
  >
    <img
      alt=""
      className="landing_image"
      src="home.png"
    />
    <div>
      <div
        className="mobile_navigation"
        data-active={false}
        data-is-login={false}
      >
        <div
          className="mobile_navigation_top_side"
        >
          <svg
            className="mobile_navigation_grid_view"
            height="32"
            style={
              Object {
                "height": "1em",
                "verticalAlign": "middle",
                "width": "auto",
              }
            }
            viewBox="0 0 32 32"
            width="32"
          >
            <path
              className="cls-1"
              d="M3.455,4A1.491,1.491,0,0,0,2,5.684v5.053a1.491,1.491,0,0,0,1.455,1.684H9.273a1.491,1.491,0,0,0,1.455-1.684V5.684A1.491,1.491,0,0,0,9.273,4ZM15.091,4a1.491,1.491,0,0,0-1.455,1.684v5.053a1.491,1.491,0,0,0,1.455,1.684h5.818a1.491,1.491,0,0,0,1.455-1.684V5.684A1.491,1.491,0,0,0,20.909,4ZM26.727,4a1.491,1.491,0,0,0-1.455,1.684v5.053a1.491,1.491,0,0,0,1.455,1.684h5.818A1.491,1.491,0,0,0,34,10.737V5.684A1.491,1.491,0,0,0,32.545,4ZM3.455,15.789A1.491,1.491,0,0,0,2,17.474v5.053a1.491,1.491,0,0,0,1.455,1.684H9.273a1.491,1.491,0,0,0,1.455-1.684V17.474a1.491,1.491,0,0,0-1.455-1.684Zm11.636,0a1.491,1.491,0,0,0-1.455,1.684v5.053a1.491,1.491,0,0,0,1.455,1.684h5.818a1.491,1.491,0,0,0,1.455-1.684V17.474a1.491,1.491,0,0,0-1.455-1.684Zm11.636,0a1.491,1.491,0,0,0-1.455,1.684v5.053a1.491,1.491,0,0,0,1.455,1.684h5.818A1.491,1.491,0,0,0,34,22.526V17.474a1.491,1.491,0,0,0-1.455-1.684ZM3.455,27.579A1.491,1.491,0,0,0,2,29.263v5.053A1.491,1.491,0,0,0,3.455,36H9.273a1.491,1.491,0,0,0,1.455-1.684V29.263a1.491,1.491,0,0,0-1.455-1.684Zm11.636,0a1.491,1.491,0,0,0-1.455,1.684v5.053A1.491,1.491,0,0,0,15.091,36h5.818a1.491,1.491,0,0,0,1.455-1.684V29.263a1.491,1.491,0,0,0-1.455-1.684Zm11.636,0a1.491,1.491,0,0,0-1.455,1.684v5.053A1.491,1.491,0,0,0,26.727,36h5.818A1.491,1.491,0,0,0,34,34.316V29.263a1.491,1.491,0,0,0-1.455-1.684Z"
              id="grid_ic"
              transform="translate(-2 -4)"
            />
          </svg>
          <div
            className="mobile_navigation_top_right_side"
          >
            <span
              className="mobile_navigation_title"
            >
              تیکسی
            </span>
            <div
              className="mobile_navigation_activator"
              onClick={[Function]}
            >
              <span
                className="mobile_navigation_activator_top_line"
              />
              <span
                className="mobile_navigation_activator_middle_line"
              />
              <span
                className="mobile_navigation_activator_bottom_line"
              />
            </div>
          </div>
        </div>
        <div
          className="mobile_navigation_drop_down"
        >
          <a
            className="mobile_navigation_drop_down_item"
            href="/sign-up"
            onClick={[Function]}
          >
            ثبت‌نام
          </a>
          <a
            className="mobile_navigation_drop_down_item"
            href="/sign-in"
            onClick={[Function]}
          >
            ورود
          </a>
          <div
            className="mobile_navigation_drop_down_item"
          >
            ارتباط با ما
          </div>
        </div>
      </div>
      <div
        className="navigation"
      >
        <div
          className="link"
        >
          <div
            onClick={[Function]}
          >
            ارتباط با ما
          </div>
        </div>
        <div
          className="link"
        >
          <a
            data-testid="sign-in-button"
            href="/sign-in"
            onClick={[Function]}
          >
            ورود
          </a>
          <span>
            /
          </span>
          <a
            data-testid="sign-up-button"
            href="/sign-up"
            onClick={[Function]}
          >
            ثبت‌نام
          </a>
        </div>
        <div
          className="link"
          style={
            Object {
              "color": "#103cb7",
              "fontWeight": "bold",
            }
          }
        >
          تیکسی
          <svg
            fill="#0f67cb"
            height={36}
            id="Capa_1"
            version="1.1"
            viewBox="0 0 360.49 360.49"
            width={36}
            x="0px"
            xmlns="http://www.w3.org/2000/svg"
            y="0px"
          >
            <g>
              <g>
                <g>
                  <path
                    d="M96.653,0H13.061C7.29,0,2.612,4.678,2.612,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449V10.449C107.102,4.678,102.424,0,96.653,0z"
                  />
                  <path
                    d="M222.041,0h-83.592C132.678,0,128,4.678,128,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449V10.449C232.49,4.678,227.812,0,222.041,0z"
                  />
                  <path
                    d="M96.653,125.388H13.061c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C107.102,130.066,102.424,125.388,96.653,125.388z"
                  />
                  <path
                    d="M222.041,125.388h-83.592c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C232.49,130.066,227.812,125.388,222.041,125.388z"
                  />
                  <path
                    d="M347.429,0h-83.592c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449V10.449C357.878,4.678,353.199,0,347.429,0z"
                  />
                  <path
                    d="M347.429,125.388h-83.592c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C357.878,130.066,353.199,125.388,347.429,125.388z"
                  />
                  <path
                    d="M96.653,256H13.061c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C107.102,260.678,102.424,256,96.653,256z"
                  />
                  <path
                    d="M222.041,256h-83.592c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C232.49,260.678,227.812,256,222.041,256z"
                  />
                  <path
                    d="M347.429,256h-83.592c-5.771,0-10.449,4.678-10.449,10.449v83.592c0,5.771,4.678,10.449,10.449,10.449h83.592
 c5.771,0,10.449-4.678,10.449-10.449v-83.592C357.878,260.678,353.199,256,347.429,256z"
                  />
                </g>
              </g>
            </g>
          </svg>
          <img
            alt="logo"
            src="ticksy.png"
          />
        </div>
      </div>
      <div
        className="title"
      >
        <p>
          سامانه 
          <span>
            تیکتینگ
          </span>
        </p>
        <p>
          دانشگاه خوارزمی
        </p>
        <p
          dir="auto"
        >
          همه ما بارها و بار ها شده که به مسائل مختلف در دانشگاه بر بخوریم و نیاز داشته باشیم تا با جایی برای حل اون مشکل ارتباط بگیریم و راهی برای ارتباط با مسئول مربوطه پیدا نکنیم، حالا ما اینجا هستیم تا برای شما سامانه رو مهیا کنیم تا ارتباط پایداری بین مسئولان و دانشجویان ایجاد کنیم.
          <br />
           در ضمن اگر در دانشگاه نقشی داشته باشی و نیاز باشه تا با عده ای دانشجویان ارتباط بگیرید کافیه تا مدارکت رو از طریق پروفایل برامون بفرستید و تاپیک خودتون رو ایجاد کنید.
        </p>
      </div>
    </div>
  </div>
  <div
    className="main"
  >
    <h2
      className="main_title"
    >
      تاپیک های پیشنهادی
    </h2>
    <div
      className="topic_container"
    >
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
      <div
        className="topic"
        onClick={[Function]}
      >
        <div
          className="topic_badge"
        >
          <img
            alt=""
            src="topic_badge.png"
          />
        </div>
        <div
          className="topic_content"
        >
          <h3>
            لورم اپسیوم متن ساختگی
          </h3>
          <h4>
            نیاز و کاربردهای متنوع با هدف بهبود ابزار کاربردی میباشد
          </h4>
        </div>
        <div
          className="topic_pic"
        >
          <img
            alt=""
            src="topic_icon.png"
          />
        </div>
      </div>
    </div>
  </div>
  <footer
    className="footer"
    id="contact-us"
  >
    <div
      className="footer_contact"
    >
      <p>
        <img
          alt="logo"
          src="ticksy.png"
        />
        سامانه تیکتینگ
      </p>
      <div>
        <svg
          fill="#26a9ad"
          id="Capa_1"
          viewBox="0 0 460.8 460.8"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
        >
          <g>
            <g>
              <path
                d="M230.432,0c-65.829,0-119.641,53.812-119.641,119.641s53.812,119.641,119.641,119.641s119.641-53.812,119.641-119.641
 S296.261,0,230.432,0z"
              />
            </g>
          </g>
          <g>
            <g>
              <path
                d="M435.755,334.89c-3.135-7.837-7.314-15.151-12.016-21.943c-24.033-35.527-61.126-59.037-102.922-64.784
 c-5.224-0.522-10.971,0.522-15.151,3.657c-21.943,16.196-48.065,24.555-75.233,24.555s-53.29-8.359-75.233-24.555
 c-4.18-3.135-9.927-4.702-15.151-3.657c-41.796,5.747-79.412,29.257-102.922,64.784c-4.702,6.792-8.882,14.629-12.016,21.943
 c-1.567,3.135-1.045,6.792,0.522,9.927c4.18,7.314,9.404,14.629,14.106,20.898c7.314,9.927,15.151,18.808,24.033,27.167
 c7.314,7.314,15.673,14.106,24.033,20.898c41.273,30.825,90.906,47.02,142.106,47.02s100.833-16.196,142.106-47.02
 c8.359-6.269,16.718-13.584,24.033-20.898c8.359-8.359,16.718-17.241,24.033-27.167c5.224-6.792,9.927-13.584,14.106-20.898
 C436.8,341.682,437.322,338.024,435.755,334.89z"
              />
            </g>
          </g>
        </svg>
        <span>
          supourtickcy@khu.com
        </span>
      </div>
      <div>
        <svg
          fill="#26a9ad"
          id="Layer_1"
          version="1.1"
          viewBox="0 0 512 512"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
        >
          <g>
            <g>
              <path
                d="M256,0C153.755,0,70.573,83.182,70.573,185.426c0,126.888,165.939,313.167,173.004,321.035
 c6.636,7.391,18.222,7.378,24.846,0c7.065-7.868,173.004-194.147,173.004-321.035C441.425,83.182,358.244,0,256,0z M256,469.729
 c-55.847-66.338-152.035-197.217-152.035-284.301c0-83.834,68.202-152.036,152.035-152.036s152.035,68.202,152.035,152.035
 C408.034,272.515,311.861,403.37,256,469.729z"
              />
            </g>
          </g>
          <g>
            <g>
              <path
                d="M256,92.134c-51.442,0-93.292,41.851-93.292,93.293s41.851,93.293,93.292,93.293s93.291-41.851,93.291-93.293
 S307.441,92.134,256,92.134z M256,245.328c-33.03,0-59.9-26.871-59.9-59.901s26.871-59.901,59.9-59.901s59.9,26.871,59.9,59.901
 S289.029,245.328,256,245.328z"
              />
            </g>
          </g>
        </svg>
         تهران - خیابان شهید مفتح نرسیده به انقلاب پلاک ۴۳
      </div>
      <div>
        <svg
          fill="#26a9ad"
          id="Layer_1"
          version="1.1"
          viewBox="0 0 512 512"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
        >
          <g>
            <g>
              <path
                d="M256,0C153.755,0,70.573,83.182,70.573,185.426c0,126.888,165.939,313.167,173.004,321.035
 c6.636,7.391,18.222,7.378,24.846,0c7.065-7.868,173.004-194.147,173.004-321.035C441.425,83.182,358.244,0,256,0z M256,469.729
 c-55.847-66.338-152.035-197.217-152.035-284.301c0-83.834,68.202-152.036,152.035-152.036s152.035,68.202,152.035,152.035
 C408.034,272.515,311.861,403.37,256,469.729z"
              />
            </g>
          </g>
          <g>
            <g>
              <path
                d="M256,92.134c-51.442,0-93.292,41.851-93.292,93.293s41.851,93.293,93.292,93.293s93.291-41.851,93.291-93.293
 S307.441,92.134,256,92.134z M256,245.328c-33.03,0-59.9-26.871-59.9-59.901s26.871-59.901,59.9-59.901s59.9,26.871,59.9,59.901
 S289.029,245.328,256,245.328z"
              />
            </g>
          </g>
        </svg>
        کرج - انتهای خیابان شهید بهشتی - میدان دانشگاه
         
      </div>
    </div>
    <div
      className="footer_social"
    >
      <div>
        <img
          alt="facebook"
          src="facebook.svg"
        />
      </div>
      <div>
        <img
          alt="instagram"
          src="instagram.svg"
        />
      </div>
      <div>
        <img
          alt="linkedin"
          src="linkedin.svg"
        />
      </div>
      <div>
        <img
          alt="twitter"
          src="twitter.svg"
        />
      </div>
    </div>
  </footer>
</div>
`;
